### PR TITLE
Initial readme and contributing docs

### PR DESCRIPTION
**Pull request by reyang**
as title

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  as `work-in-progress`, or mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+* Make sure CLA is signed and CI is clear.
+
+### How to Get PRs Merged
+
+A PR is considered to be **ready to merge** when:
+* It has received two approvals from [Approvers](https://github.com/open-telemetry/community/blob/master/community-membership.md#approver)
+  / [Maintainers](https://github.com/open-telemetry/community/blob/master/community-membership.md#maintainer)
+  (at different companies).
+* Major feedbacks are resolved.
```
Comment (z1c0) [inline CONTRIBUTING.md]: Major feedback items/points are resolved.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* It has been open for review for at least one working day. This gives people
+  reasonable time to review.
+* Trivial change (typo, cosmetic, doc, etc.) doesn't have to wait for one day.
```
Comment (z1c0) [inline CONTRIBUTING.md]: Trivial changes (typo, cosmetic, doc, etc.) don't have to wait for one day.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+* Urgent fix can take exception as long as it has been actively communicated.
```
Comment (z1c0) [inline CONTRIBUTING.md]: Urgent fixes can take exceptions as long as it has been actively communicated.

File: CONTRIBUTING.md (hunk 1 above)
Comment (reyang) [reply CONTRIBUTING.md]: Thanks! Fixed.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+TBD
+
+## Contributing
+
+See [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Release Schedule
+
+OpenTelemetry C/C++ is under active development.
```
Comment (meastp) [inline README.md]: Is this supposed to be a combined library for both C and C++ users? Are C users/libraries willing to depend on a C++ runtime for their libraries/products? If this is not the case (which I hope), please do not mix C into the naming, and use just "OpenTelemetry C++"

Comment (reyang) [reply README.md]: This is a "to-be-discussed" topic for our 1st community meeting.

Comment (jmacd) [reply README.md]: I could imagine the potential-future C API belongs in its own repository.

Comment (reyang) [reply README.md]: Yes, we've discussed this during the community meeting.
For now we will use this repo for the C++ SDK, and build a C API (no exception, no STL in the function signature) on top of the C++ SDK.
Later we might want to split the C API to its own repo.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+The OpenTelemetry C/C++ special interest group (SIG) meets regularly. See the
+OpenTelemetry [community](https://github.com/open-telemetry/community#cc-sdk)
+repo for information on this and other language SIGs.
+
+See the [public meeting notes](https://docs.google.com/document/d/1i1E4-_y4uJ083lCutKGDhkpi3n4_e774SBLi9hPLocw/edit)
+for a summary description of past meetings. To request edit access, join the
+meeting or get in touch on [Gitter](https://gitter.im/open-telemetry/opentelemetry-cpp).
+
+See to the [community membership document](https://github.com/open-telemetry/community/blob/master/community-membership.md)
```
Comment (g-easy) [inline CONTRIBUTING.md]: Nit: see to the -> see the

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+$ git push fork feature
+```
+
+Open a pull request against the main `opentelemetry-cpp` repo.
+
+### How to Receive Comments
+
+* If the PR is not ready for review, please put `[WIP]` in the title, tag it
+  as `work-in-progress`, or mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+* Make sure CLA is signed and CI is clear.
```
Comment (g-easy) [inline CONTRIBUTING.md]: Can we have a link for the CLA please?

File: CONTRIBUTING.md (hunk 5 above)
Comment (reyang) [reply CONTRIBUTING.md]: Nice catch, fixed.

File: CONTRIBUTING.md (hunk 6 above)
Comment (reyang) [reply CONTRIBUTING.md]: Added link CNCF CLA.
BTW, given CLA is covered as part of CI, should we just remove it and mention CI only (for sake of the doc simplicity)?

Comment (g-easy) [reply CONTRIBUTING.md]: Up to you. :)

**Pull request merged**